examples/alchemy-daapp/package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@alchemy/aa-core": "file:../../packages/core",
```

Comment:
oh... hmm what's going on here? does this not work with just `"*"` as the version?

Reply:
if we do * it'll ask for a version whenever we run yarn. Doing it this way forces the example to use the version that is specified in the core package.json.

Reply:
<img width="757" alt="Screenshot 2023-06-05 at 10 00 20 PM" src="https://github.com/alchemyplatform/aa-sdk/assets/3498866/15fdc74c-a34a-400a-af2c-f9da559fdecb">